### PR TITLE
update v1.0.0 yaml path

### PR DESCRIPTION
**Pull request by raunakkumar**
Signed-off-by: Raunak Kumar <rkumar@nimblestorage.com>

### CODE REVIEW

File: yaml/csi-driver/v1.1.0/nimble-csp.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: nimble-csp
+    spec:
+      priorityClassName: system-cluster-critical
+      containers:
+        - name: nimble-csp
+          image: hpestorage/nimble-csp:v1.0.0
```
Comment (rgcostea) [inline yaml/csi-driver/v1.1.0/nimble-csp.yaml]: Should this be 1.1.0 to match the path?  Or even "edge" for now until we spin a beta?

Comment (shivamerla) [reply yaml/csi-driver/v1.1.0/nimble-csp.yaml]: Is it better to create edge directory with YAML files pointing to edge images? with 1.1.0 directory, users might think this is the latest release and use these YAML files instead.

Comment (rgcostea) [reply yaml/csi-driver/v1.1.0/nimble-csp.yaml]: That works too.  We probably shouldn't create the 1.1.0 directory until we release it.

Comment (raunakkumar) [reply yaml/csi-driver/v1.1.0/nimble-csp.yaml]: updated the folder and tag to edge

**Pull request merged**